src/Components/Steps/ZipcodeStep.tsx

```diff
@@ -120,14 +117,20 @@ export const ZipcodeStep = () => {
 
   return (
     <div>
-      <QuestionLeadText>{matchingQuestion.subheader}</QuestionLeadText>
-      <QuestionHeader>{matchingQuestion.header}</QuestionHeader>
-      <QuestionQuestion>{matchingQuestion.question}</QuestionQuestion>
+      <QuestionLeadText>
+        <FormattedMessage id="qcc.tell-us-text" defaultMessage="Let's Get Started!" />
+      </QuestionLeadText>
```

Comment:
Works for now, but we might need to make this dynamic if this is not the first question.